x-pack/plugins/monitoring/public/lib/format_number.js

```diff
@@ -30,6 +30,8 @@ export function formatNumber(num, which) {
   switch (which) {
     case 'time_since':
       return moment(moment() - num).from(moment(), true);
+    case 'time_since2':
```

Comment:
Would be good to have a bit a more descriptive variable name here to describe the difference between `time_since` and `time_since2`.

Reply:
Yes absolutely. Once I get this in a better state with Gil, I'm going to figure out why we need this and can't just use `time_since`